_episodes/09-conflict.md

```diff
@@ -47,21 +47,22 @@ $ cat mars.txt
 Cold and dry, but everything is my favorite color
 The two moons may be a problem for Wolfman
 But the Mummy will appreciate the lack of humidity
-This line added to Wolfman's copy
+An underground base with few windows would be best, to minimise Wolfman's exposure to the moons
```

Comment:
These lines are too long.  They should be shorter, and add only one line at a time.  This allows the learner to focus on how Git operates instead of typing out all the extraneous details of the story. 

Maybe

> An underground base to minimise Wolfman's exposure to the moons

Reply:
I agree these lines are too long; I was trying to work out how to make it clear who wrote them while also making a point relevant to the story. Thanks for the suggestion of a shorter alternative.

I disagree that there should only be one line in each change. My concern is that this will give the impression that the way to deal with a merge conflict is "delete every conflicting line that Git has highlighted and start again". The vast majority of merge conflicts will involve leaving some lines of the original present; I think having the second line present makes it much clearer what the thought process behind the conflict resolution is.